lib/lettering/font.py

```diff
@@ -103,28 +108,58 @@ def _check_variants(self):
 
     name = localized_font_metadata('name', '')
     description = localized_font_metadata('description', '')
-    default_variant = font_metadata('default_variant', FontVariant.LEFT_TO_RIGHT)
```

Comment:
What are your thoughts behind getting rid of default_variant from the font metadata?  I was thinking we could just let the author decide...  is that not working well?

Reply:
We are still reading it just a bit later and check if it exists as a file.

Reply:
I see it now!